karaf/itest/src/test/java/org/apache/brooklyn/KarafTestUtils.java

```diff
@@ -44,7 +46,9 @@
         configureConsole().ignoreLocalConsole(),
         logLevel(LogLevel.INFO),
         features(karafStandardFeaturesRepository(), "eventadmin"),
-        junitBundles()
+        junitBundles(),
+        editConfigurationFilePut("etc/org.apache.brooklyn.osgilauncher.cfg", "persistMode", "AUTO"),
+        editConfigurationFilePut("etc/org.apache.brooklyn.osgilauncher.cfg", "persistenceDir", new File("target/paxexam/persistence/" + Identifiers.makeRandomId(6) + "/").getAbsolutePath()),
```

Comment:
Looks good. I presume target will get deleted at some point (e.g. on the next clean build), especially when run on local machines? So no need for tearDown to delete the folder?

With that assumption, very happy to merge. Doing so now.